### DISCUSSION TITLE
Is it possible to implement KDE's custom window decorations?

### DISCUSSION THREAD

**Discussion by BonjeBongo** [Floorp-Projects/Floorp#1302] [Feature Request]
I've tried FF Ultima's override but that didn't work and floorp still uses GTK theme

https://github.com/soulhotel/FF-ULTIMA/blob/main/theme/override-linux.css

**Comment by CutterKnife** [comment]
The current Floorp 11 is based on ESR 115; FF Ultima requires Firefox 120 or later, so that may have something to do with it.
